releasenotes/notes/313-enable-integrations-01990085769ea3f3.yaml

```diff
@@ -0,0 +1,17 @@
+---
+upgrade:
+  - |
+    Enables tests with Python 3.13 for the following integrations:
```

Comment:
```suggestion
    Documents Python 3.13 support for the following integrations:
```

Reply:
Does this not technically "enable" testing 3.13 support since there are new 3.13 environments generated?